Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -31,8 +31,8 @@
   #define MKS_MINI_12864
 #endif
 
-// MKS_MINI_12864_V3 is simply identical to FYSETC_MINI_12864_2_1
-#if ENABLED(MKS_MINI_12864_V3)
+// MKS_MINI_12864_V3 and BTT_MINI_12864_V1 are simply identical to FYSETC_MINI_12864_2_1
+#if EITHER(MKS_MINI_12864_V3, BTT_MINI_12864_V1)
   #define FYSETC_MINI_12864_2_1
```

Comment:
Some pins files specifically refer to the `MKS_MINI_12864_V3` but not to `FYSETC_MINI_12864_2_1`. It looks like these pins files should use `FYSETC_MINI_12864_2_1` in place of `MKS_MINI_12864_V3` so they may be compatible with all three aliases.

Reply:
Agreed.